platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/offline/OfflineUtils.java

```diff
@@ -0,0 +1,42 @@
+package com.mapbox.mapboxsdk.testapp.activity.offline;
+
+import android.support.annotation.NonNull;
+import android.util.Log;
+
+import com.mapbox.mapboxsdk.offline.OfflineRegion;
+
+import org.json.JSONObject;
+
+import static com.mapbox.mapboxsdk.testapp.activity.offline.OfflineActivity.JSON_CHARSET;
+import static com.mapbox.mapboxsdk.testapp.activity.offline.OfflineActivity.JSON_FIELD_REGION_NAME;
+
+class OfflineUtils {
+
+    static String getRegionName(@NonNull  OfflineRegion offlineRegion) {
```

Comment:
Let's make this method symmetric with `byte[] convertRegionName(String regionName)` below and change its signature to `String getRegionName(@NonNull byte[] metadata)`. We don't need the `OfflineRegion` reference here. After all, this is just a generic util to wrap from/to `String` <-> `JSONObject` <-> `byte[]`.